database/sqlite/sqlite_metadata.c

```diff
@@ -190,6 +197,7 @@ static int check_and_update_chart_labels(RRDSET *st, BUFFER *work_buffer, size_t
     struct query_build tmp = {.sql = work_buffer, .count = 0};
     uuid_unparse_lower(st->chart_uuid, tmp.uuid_str);
     rrdlabels_walkthrough_read(st->rrdlabels, chart_label_store_to_sql_callback, &tmp);
+    buffer_strcat(work_buffer, " ON CONFLICT (chart_id, label_key) DO UPDATE SET source_type = excluded.source_type, label_value=excluded.label_value, date_created=UNIXEPOCH()");
```

Comment:
If there is a conflict, we already had data before, so why are we naming field as `date_created`? Should we not use a more meaningful name?

Reply:
yes correct but at this point it would require a migration. Can do in a another PR (eg. last_updated or last_used)